package.json

```diff
@@ -143,13 +143,13 @@
     "comment-parser": "^1.4.1",
     "custom-elements-manifest": "^2.1.0",
     "esbuild": "^0.25.0",
-    "eslint": "^9.28.0",
+    "eslint": "^9.31.0",
```

Comment:
`eslint-plugin-unicorn` has a peer dependency of `eslint@9.29.0`. Went ahead and bumped it all the way up.